homeassistant/components/dialogflow/__init__.py

```diff
@@ -79,6 +79,11 @@ class DialogFlowError(HomeAssistantError):
     hass.components.webhook.async_unregister(entry.data[CONF_WEBHOOK_ID])
     return True
 
+
+# pylint: disable=invalid-name
+async_remove_entry = config_entry_flow.webhook_async_remove_entry
```

Comment:
I'm confused why we need to do this separately from the `async_unload_entry`. What's the difference between these two methods?

Reply:
An entry can be unloaded and then loaded again. (I still need to add this to the frontend).

Only on removal will we clean it all up.

Reply:
It's like start, stop and delete.

Reply:
Once you describe it like that, it makes complete sense... Thanks!